blueprints/ember-cli-notifications/index.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   // Use config to determine whether Font Awesome is imported into consuming app
   afterInstall: function(app) {
     var projectConfig = this.project.config(app.env);
-    var config = projectConfig['ember-cli-notifications'];
+    var config = projectConfig['ember-cli-notifications'] ||
+                 app.options['ember-cli-notifications'] ||
+                 { includeFontAwesome: false };
```

Comment:
By looking at the 2.0.0 code [here](https://github.com/Blooie/ember-cli-notifications/blob/v2.0.0/index.js#L13) it seems that it doesn't include `font-awesome` by default. Also the [README](https://github.com/Blooie/ember-cli-notifications/blob/v2.0.0/README.md#font-awesome) states that.

I believe that this should be considered a regression and left `false` by default to avoid breaking SemVer so that existing apps that update don't have errors.

On the other hand I understand that it would be nice for the addon to work out of the box when installed stand-alone. I don't think that asking the user to define `'includeFontAwesome': true` or having an extra `ember install ember-cli-font-awesome` is too much to ask for a developer and if it's clear in the README then I don't see much of a problem there.